### DISCUSSION TITLE
"This user account is blocked"

### DISCUSSION THREAD

**Discussion by Amir-Asgari** [vercel/community#3916] [Account Issue] [answered]
### Question

I encountered the error "This user has been blocked" during registration. Can you check the problem?
I have not registered on this site yet.
email: mr.a.asgarigmail.com 
GitHub : https://github.com/Amir-Asgari

**Comment by swarnava** [comment, marked as answer]
Hi, 

Please email us at registration@vercel.com

**Comment by Amir-Asgari** [comment]
Hi
I sent an email to you 3 days ago, but I didn't hear back from you.
